cmd/goat/account_migrate.go

```diff
@@ -5,16 +5,15 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"log/slog"
-	"strings"
-	"time"
-
 	"github.com/bluesky-social/indigo/api/agnostic"
 	comatproto "github.com/bluesky-social/indigo/api/atproto"
 	"github.com/bluesky-social/indigo/atproto/syntax"
 	"github.com/bluesky-social/indigo/xrpc"
-
+	"github.com/manifoldco/promptui"
```

Comment:
do you think we really need this new dependency, or could we just use the `bufio.NewReader(os.Stdin)` pattern?

Reply:
Thank you for the feedback! I agree with everything else. I had used this dependency since I saw it used in another cli, and gave some nice quality of life things (auto y/N, styling, one liner, etc), and I am not as familiar with Go's apis. But I will take some time this week to give it a try with `bufio.NewReader(os.Stdin)` and see what that looks like.